internal/grafanadi/handler/debugpodlist.go

```diff
@@ -0,0 +1,105 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/alipay/container-observability-service/internal/grafanadi/service"
+	"github.com/alipay/container-observability-service/pkg/dal/storage-client/data_access"
+	"github.com/alipay/container-observability-service/pkg/dal/storage-client/model"
+	"github.com/alipay/container-observability-service/pkg/metrics"
+	"github.com/alipay/container-observability-service/pkg/utils"
+)
+
+type DebugPodListHandler struct {
```

Comment:
The word DebugPodList is misleading, QueryPodList maybe better.

Reply:
fixed